.eslintrc

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
currently using `var` is not flag by eslint rename this file from `.eslintrc` to `.eslintrc.json` .

Reply:
Alright

Reply:
Done